pkg/download/download.go

```diff
@@ -38,24 +43,31 @@ func NewClient(opts *ClientOpts) (*Client, error) {
 		}
 	}
 
+	// the client needs either routing number && download code OR download URL
+
 	if opts.RoutingNumber == "" {
-		return nil, errors.New("missing routing number")
+		return nil, errors.New(MissingRoutingNumberErr)
 	}
 	if opts.DownloadCode == "" {
-		return nil, errors.New("missing download code")
+		return nil, errors.New(MissingDownloadCodeErr)
+	}
+
+	if opts.DownloadURL == "" {
+		opts.DownloadURL = "https://frbservices.org/EPaymentsDirectory/directories/%s?format=json"
```

Comment:
Would it make sense to set this URL as a constant somewhere, or be pulled from configuration?

Reply:
We can make it a constant. This project doesn't have a config file to read from.